templates/getting_started/tensorflow_keras_versions.md

```diff
@@ -0,0 +1,87 @@
+# TensorFlow 2.16 with Keras
```

Comment:
This whole page is very version-specific, which means it will get outdated very quickly. Could we instead edit the "TF compatibility" section on https://keras.io/getting_started/ ?

Reply:
Agree.  Will update the Getting Started page and create a new PR